### ISSUE TITLE
Broken code blocks in SAW tutorial

### ISSUE THREAD

**Issue by qsctr** [GaloisInc/saw-script#1900]
It seems like something went wrong with the SAW website build process since https://saw.galois.com/tutorial.html now shows things like `$include 9-17 code/ffs.c` instead of the actual code to be included. I remember it being fine a few weeks ago, but the tutorial part of the docs doesn't seem to have been touched in the last few weeks so not sure what the problem is.

![image](https://github.com/GaloisInc/saw-script/assets/10395793/0da64142-1ab7-468f-a4a4-c4fd116229f9)

**Comment by RyanGlScott** [comment]
I believe this is my doing, albeit by accident. I updated the SAW website fairly recently, and while I thought updating the SAW tutorial was a simple matter of copying over [`tutorial.md`](https://github.com/GaloisInc/saw-script/blob/2011a770479d98edd78d512dd6a7242525223a3b/doc/tutorial/tutorial.md), it turns out that the process is more involved than I realized. I'll work on fixing the website soon.
